GITHUB-TEAMS.md

```diff
@@ -0,0 +1,18 @@
+In order to ease notifying folks with a common expertise or interest, WHATWG uses GitHub teams. WHATWG uses teams in the following manner:
+
+* Anyone can join any team they desire. To join just let someone know or [file an issue](https://github.com/whatwg/meta/issues/new).
+* Everyone tries to ensure the appropriate teams are pinged on relevant issues/pull requests.
+* Teams have no formal authority, though it's fine to add them as contributors to repositories for ease of collaboration. For all repositories we care about protecting master primarily. The bar for write access to other branches is much much lower.
+* Teams have no responsibility either. If you ping a team and get no feedback, moving things forward responsibly is still on you.
+* Creating new teams is as simple as [filing an issue](https://github.com/whatwg/meta/issues/new). Review should take care it is not too much of a niche or overlap with existing teams.
```

Comment:
It's not that simple; it needs to get created by an admin. Maybe "To propose a new team, file an issue. We'll approve most teams unless they are a niche or overlap with an existing team."